index.html

```diff
@@ -36,17 +38,19 @@
     <header>
         <nav>
             <a href="index.html" id="logonav">Jhon Doe</a>
-            <img id="menuimg" src="bars-solid.svg" alt="menu icon">
-            <ul class="fullmenu">
-                <li class="menuitem">Portafolio</li>
-                <li class="menuitem">About</li>
-                <li class="menuitem">Contact</li>
-                <li id="envelope" class="menuitem"><i class="fa-regular fa-envelope"></i></li>
+            <img id="menuimg" class="menulogo" src="bars-solid.svg" alt="menu icon">
+            <a id="closingbutton" href=""><i class="fa-solid fa-xmark"></i></a>
+            <ul id="fullmenu" class="displaymenu">
+                
+                <li id="menuitemport" class="menuitem"><a href="#welcomesection">Portfolio</a></li>
+                <li id="menuitemabout" class="menuitem"><a href="#aboutmeid">About</a></li>
+                <li id="menuitemcontact" class="menuitem"><a href="#contactmeid">Contact</a></li>
+                <li id="envelope" class="menuitem"><a href="#contactmeid"><i class="fa-regular fa-envelope"></i></a></li>
             </ul>
```

Comment:
### Your Project
![desktop-view (copy)](https://user-images.githubusercontent.com/93039359/179865618-fdb1b325-9bd8-456a-a2cc-2f6aa81ad87d.png)

- [ ] Your logo should be responsible to take back to the top and your Portfolio link should be responsible to navigate the work section of your portfolio :blush: